distribution/docker/solr/src/main/resources/docker-ddf-solr-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This entrypoint script will be called first to do some ddf specific things before handing off to the official solr docker-entrypoint script
+
+if [ ! -z "${CORES}" ]; then
```

Comment:
❓ Where is the actual list of cores defined? Is it a part of this PR?

Reply:
Six months ago when we started pulling out Solr, there was an Itest somewhere that dynamically created a core. I don't know if that test still exists. Downstream projects might also create cores as part of what they do. Maybe we should think of a way to support dynamic core creation in docker-ized Solr?  In fact, I wonder how hard it would be to create a Solr extension that creates cores?